src/main/java/com/powsybl/openloadflow/network/NetworkState.java

```diff
@@ -47,14 +50,16 @@ public static NetworkState save(LfNetwork network) {
         List<BusState> busStates = ElementState.save(network.getBuses(), BusState::save);
         List<BranchState> branchStates = ElementState.save(network.getBranches(), BranchState::save);
         List<HvdcState> hvdcStates = ElementState.save(network.getHvdcs(), HvdcState::save);
-        return new NetworkState(network, busStates, branchStates, hvdcStates, network.getExcludedSlackBuses());
+        List<AreaState> areaStates = ElementState.save(network.getAreaStream().toList(), AreaState::save);
```

Comment:
would be cleaner that LfArea extends cleanly AbstractElement and that we get a new network.getAreas() method backed by a List<LfArea>

Reply:
Done, did not see AbstractElement was there.